### ISSUE TITLE
VRRP VIP not configured on device when VRRP section is marked as "optional"

### ISSUE THREAD

**Issue by cmohorea** [CiscoDevNet/terraform-provider-sdwan#213]
When creating VRRP section as "optional", I noticed that VIP IP would not show up in the resulting device config (see below). 
When removing the "optional" part, it goes into config. SD-WAN 20.9/17.9

```
resource "sdwan_cisco_vpn_interface_feature_template" "SITE_LAN_SVPN_FSERVICES_SUB200" {
[...skipped...]
  ipv4_vrrps =   [
    {
      optional = true,
      group_id_variable = "SVPN_FSERVICES_LAN_SUB200_VRRP_GRP",
      ip_address_variable = "SVPN_FSERVICES_LAN_SUB200_VRRP_VIP_ADDR",
      priority_variable = "SVPN_FSERVICES_LAN_SUB200_VRRP_PRI",
      track_prefix_list = "DEFAULT_ROUTE",
    }
  ]
}
```
results in
```
interface TenGigabitEthernet0/0/5.200
[...skipped...]
 vrrp 100 address-family ipv4
  timers advertise 100
  priority 90
  vrrpv2
  track 1 shutdown
  exit-vrrp
 arp timeout 1200
end
```

However, 
```
resource "sdwan_cisco_vpn_interface_feature_template" "SITE_LAN_SVPN_FSERVICES_SUB100" {
[...skipped...]
  ipv4_vrrps =   [
    {
      group_id = "100",
      ip_address_variable = "SVPN_FSERVICES_LAN_SUB100_VRRP_VIP_ADDR",
      priority_variable = "SVPN_FSERVICES_LAN_SUB100_VRRP_PRI",
      track_prefix_list = "DEFAULT_ROUTE",
    }
  ]
}
```
results in
```
interface TenGigabitEthernet0/0/5.100
[...skipped...]
 vrrp 100 address-family ipv4
  timers advertise 100
  priority 90
  vrrpv2
  track 1 shutdown
>>address 10.45.147.129 primary
  exit-vrrp
 arp timeout 1200
end
```

**Comment by danischm** [comment]
https://github.com/CiscoDevNet/terraform-provider-sdwan/commit/69ad80b9596a21686b7dd35cd6ddda33923c4866

**Comment by danischm** [comment]
Fix added in v0.3.8 .
